hue/hue.sh

```diff
@@ -53,8 +54,20 @@ function update_repo() {
   if command -v apt-get >/dev/null; then
     retry_command "apt-get update"
   else
-    retry_command "yum -y update"
```

Comment:
if '-y' causing issue, can we add comment why not use it.

Reply:
Good catch. I should not have removed the `-y`.